web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -554,24 +554,24 @@ import {get_string_diff} from "../../src/util";
                     ) {
                         return;
                     }
-                    var hideOnEmpty = false;
```

Comment:
Adding braces and using `let` would have been another way to solve `no-case-declarations`, though I don’t object to this solution.